src/Models/BaseModelInterface.php

```diff
@@ -0,0 +1,7 @@
+<?php
+namespace Afonso\LvCrud\Models;
+
+interface BaseModelInterface
+{
+	public function getValidationRules();
```

Comment:
This function needs documenting, as well as the interface.

I think the old name `BaseModel` wasn't quite right. How about naming this interface `CrudModelInterface`? Developers might already have some other sort of base model.